<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20984" started-at="2018-05-21T15:02:44Z" finished-at="2018-05-21T15:03:05Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20984" started-at="2018-05-21T15:02:44Z" finished-at="2018-05-21T15:03:05Z">
      <class name="automateWin.Excel">
        <test-method status="FAIL" signature="initialize()[pri:0, instance:automateWin.Excel@11758f2a]" name="initialize" is-config="true" duration-ms="20314" started-at="2018-05-21T20:32:44Z" finished-at="2018-05-21T20:33:05Z">
          <exception class="java.lang.IllegalStateException">
            <message>
              <![CDATA[The process has not exited yet therefore no result is available ...]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalStateException: The process has not exited yet therefore no result is available ...at org.apache.commons.exec.DefaultExecuteResultHandler.getException(DefaultExecuteResultHandler.java:76)at org.openqa.selenium.os.UnixProcess.checkForError(UnixProcess.java:171)at org.openqa.selenium.os.CommandLine.checkForError(CommandLine.java:185)at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:177)at org.openqa.selenium.remote.service.DriverService.start(DriverService.java:166)at automateWin.Reusable.invokeApplication(Reusable.java:56)at automateWin.Excel.initialize(Excel.java:15)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:163)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="SKIP" signature="initiate()[pri:0, instance:automateWin.Excel@11758f2a]" name="initiate" duration-ms="0" started-at="2018-05-21T20:33:05Z" finished-at="2018-05-21T20:33:05Z">
          <exception class="java.lang.IllegalStateException">
            <message>
              <![CDATA[The process has not exited yet therefore no result is available ...]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalStateException: The process has not exited yet therefore no result is available ...at org.apache.commons.exec.DefaultExecuteResultHandler.getException(DefaultExecuteResultHandler.java:76)at org.openqa.selenium.os.UnixProcess.checkForError(UnixProcess.java:171)at org.openqa.selenium.os.CommandLine.checkForError(CommandLine.java:185)at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:177)at org.openqa.selenium.remote.service.DriverService.start(DriverService.java:166)at automateWin.Reusable.invokeApplication(Reusable.java:56)at automateWin.Excel.initialize(Excel.java:15)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:163)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initiate -->
      </class> <!-- automateWin.Excel -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
